etc/docker/elyra/Dockerfile.dev

```diff
@@ -15,28 +15,68 @@
 # limitations under the License.
 #
 
-# Ubuntu 18.04 LTS - Bionic
+# Ubuntu 20.04.2 LTS (Focal Fossa)
 # Repository: https://hub.docker.com/r/jupyterhub/k8s-singleuser-sample/tags
 FROM jupyterhub/k8s-singleuser-sample:1.2.0
 
-
 ARG TAG="dev"
```

Comment:
NIT: the specified arg is not used and should be removed

Reply:
@ptitzler do you want me to add a commit that removes these?

Reply:
@thesuperzapper  - yes please and thank you!

Reply:
Fixed in https://github.com/elyra-ai/elyra/pull/2344/commits/592afbb1955024f2f3d32d8fcb9423b08c343d28